scripts/actions/print-matrix.clj

```diff
@@ -1,41 +1,47 @@
 #!/usr/bin/env bb
 
-(def default-clojure-version "") ;; set in project.clj
 (def default-java-version "11.0.9")
 (def java-15-version "15")
-(def clojure-next "1.10.2-rc1")
 (def non-cron-ctia-nsplits
   "Job parallelism for non cron tests."
   10)
 (def cron-ctia-nsplits
   "Job parallelism for cron tests."
   2)
 
+(defn splits-for [base nsplits]
+  {:post [(= (range nsplits)
+             (map :this_split %))
+          (= #{nsplits}
+             (into #{} (map :total_splits %)))]}
+  (for [this-split (range nsplits)]
+    (assoc base
+           :this_split this-split
+           :total_splits nsplits)))
+
```

Comment:
This code has been bugging me because mistakes here means tests being skipped. I added sufficient post conditions so I can sleep at night.